application/tests/web_main_test.py

```diff
@@ -77,15 +77,18 @@ def test_extend_cre_with_tag_links(self) -> None:
                 )
             ),
             "cc": cres["cc"],
-            "cd": cres["cd"]
-            .add_link(
-                defs.Link(
-                    ltype=defs.LinkTypes.PartOf, document=cres["ca"].shallow_copy()
+            "cd": (
```

Comment:
are the changes in this file necessary or relevant to the issue being worked on?